packages/vector_graphics/lib/src/listener.dart

```diff
@@ -127,4 +127,19 @@ class FlutterVectorGraphicsListener extends VectorGraphicsCodecListener {
     _paths.add(path);
     _currentPath = path;
   }
+
+  @override
+  void onOpacityLayer(double value) {
```

Comment:
Probably better to just have this take a paint too

Reply:
We should definitely have a paint here.

Reply:
Done